Library/PaginatorTests.swift

```diff
@@ -18,6 +18,7 @@ struct ConcreteError: Error {}
 final class PaginatorTests: XCTestCase {
   let valuesFromEnvelope: (TestEnvelope) -> [Int] = { $0.values }
   let cursorFromEnvelope: (TestEnvelope) -> Int? = { $0.cursor }
+  let totalFromEnvelope: (TestEnvelope) -> Int? = { _ in 42 }
```

Comment:
This feels weird to me. Would it make more sense to add the total to the `TestEnvelope` itself? Or maybe just add a "Hardcoded because the total won't change during a fetch" comment if that's why it's hardcoded?

Reply:
Updated tests to not use a hardcoded value